### PR TITLE
[FSDE-30972] Fix new Vue documentation

### PR DESCRIPTION
**Pull request by josecarlosgonzalezv**
https://ctl.atlassian.net/browse/FSDE-30972

### CODE REVIEW

**Review by jllr — APPROVED** [review]

File: src/website/assets/scripts/chiVersionCheck.js [hunk 1]
```diff
@@ -42,20 +53,20 @@ function fillDropdown() {
...
+    }
 
-      if (Number(v1parts[i]) == Number(v2parts[i])) {
-          continue;
-      } else if (Number(v1parts[i]) > Number(v2parts[i])) {
-          return false;
-      } else {
-          return true;
-      }
+    if (Number(v1parts[i]) == Number(v2parts[i])) {
```
Comment (SergioQCL) [inline src/website/assets/scripts/chiVersionCheck.js:63]:
```suggestion
    if (Number(v1parts[i]) === Number(v2parts[i])) {
```

File: src/website/assets/scripts/chiVersionCheck.js [hunk 2]
```diff
@@ -30,9 +38,12 @@ function fillDropdown() {
...
 
-    versionAnchor.setAttribute("href", "https://assets.ctl.io/chi/"+versions[version]);
-    versionAnchor.setAttribute("class", "chi-dropdown__menu-item");
-    versionAnchor.innerText = "v" + versions[version];
+    versionAnchor.setAttribute(
+      'href',
+      `https://${hostname}/chi/${versions[version]}`
+    );
+    versionAnchor.setAttribute('class', 'chi-dropdown__menu-item');
+    versionAnchor.innerText = 'v' + versions[version];
```
Comment (SergioQCL) [inline src/website/assets/scripts/chiVersionCheck.js:46]:
```suggestion
    versionAnchor.innerText = `v${versions[version]}`;
```

File: src/website/assets/scripts/chiVersionCheck.js [hunk 3]
```diff
@@ -42,20 +53,20 @@ function fillDropdown() {
...
 function versionCompare(v1, v2) {
   var v1parts = v1.split('.'),
-      v2parts = v2.split('.');
+    v2parts = v2.split('.');
 
   for (var i = 0; i < v1parts.length; ++i) {
-      if (v2parts.length == i) {
-          return false;
-      }
+    if (v2parts.length == i) {
```
Comment (SergioQCL) [inline src/website/assets/scripts/chiVersionCheck.js:59]:
```suggestion
    if (v2parts.length === i) {
```

File: src/website/assets/scripts/chiVersionCheck.js [hunk 4]
```diff
@@ -42,20 +53,20 @@ function fillDropdown() {
...
-      } else {
-          return true;
-      }
+    if (Number(v1parts[i]) == Number(v2parts[i])) {
+      continue;
+    } else if (Number(v1parts[i]) > Number(v2parts[i])) {
+      return false;
+    } else {
+      return true;
+    }
```
Comment (SergioQCL) [inline src/website/assets/scripts/chiVersionCheck.js:69]:
```suggestion
    } else {
      return Number(v1parts[i]) <= Number(v2parts[i]);
    }
```

**Comment by jllr** [comment]
RKTN!

**Review by SergioQCL — APPROVED** [review]
PLNT! but there are some little improvements that can be done in the future

**Pull request merged**